src/frontend/screens/Settings/components/WineExtensions/index.tsx

```diff
@@ -1,12 +1,51 @@
-import React from 'react'
+import React, { useContext } from 'react'
 import { useTranslation } from 'react-i18next'
 import BattlEyeRuntime from './BattlEyeRuntime'
 import EacRuntime from './EacRuntime'
 import AutoDXVK from './AutoDXVK'
 import AutoVKD3D from './AutoVKD3D'
+import useSetting from 'frontend/hooks/useSetting'
+import { WineInstallation } from 'common/types'
+import { defaultWineVersion } from '../WineSettings'
+import classNames from 'classnames'
+import SettingsContext from '../../SettingsContext'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons'
 
 export default function WineExtensions() {
   const { t } = useTranslation()
+  const [wineVersion] = useSetting<WineInstallation>(
+    'wineVersion',
+    defaultWineVersion
+  )
+  const { appName, runner } = useContext(SettingsContext)
+
+  if (wineVersion.type === 'bottles') {
+    return (
+      <>
+        <h3 className="settingsSubheader">
+          {t('settings.navbar.wineExt', 'Wine Extensions')}
+        </h3>
+
+        <div className="infoBox saves-warning">
+          <FontAwesomeIcon icon={faExclamationTriangle} color={'yellow'} />
+          {t(
+            'settings.wineExt.bottlesInformation',
+            'You can configure things like DXVK, VKD3D and more, in Bottles UI'
```

Comment:
```suggestion
            'You can configure DXVK, VKD3D and more, in Bottles UI'
```

Just a suggestion, we are already saying `and more` so we don't need the `things` and it may make translations easier.